addon/webextension/background/senderror.js

```diff
@@ -92,20 +102,26 @@ this.senderror = (function() {
       return;
     }
     if (!Raven.isSetup()) {
-      Raven.config(dsn, {allowSecretKey: true}).install();
+      Raven.config(dsn, {allowSecretKey: true, dataCallback: normalizeURL}).install();
     }
-    let exception = new Error(e.message);
+    // NOTE: don't use e.message, as that is a localized string
```

Comment:
e.message shouldn't be localized.  popupMessage is used to get a localized string from elsewhere for the popup.